### PR TITLE
fix(client): render live draft procedure values

### PR DESCRIPTION
**Pull request by mwd1234**
## Summary

Render live draft and sealed procedure instructions from the engine-published pack count, pack sizes, and minimum deck size instead of default-format constants. Quick, pod, and Commander intros now cover uniform and mixed-size pack procedures, while sealed copy reads the published procedure count. The localized copy composes context-specific CLDR quantity phrases and locale-aware lists across all seven catalogs, including singular and mixed-size edge cases.

## Files changed

- `client/src/components/draft/DraftIntro.tsx`
- `client/src/components/draft/SealedPackOpening.tsx`
- `client/src/components/draft/__tests__/DraftIntro.test.tsx`
- `client/src/components/draft/__tests__/SealedPackOpening.test.tsx`
- `client/src/i18n/__tests__/localeParity.test.ts`
- `client/src/i18n/locales/de/draft.json`
- `client/src/i18n/locales/en/draft.json`
- `client/src/i18n/locales/es/draft.json`
- `client/src/i18n/locales/fr/draft.json`
- `client/src/i18n/locales/it/draft.json`
- `client/src/i18n/locales/pl/draft.json`
- `client/src/i18n/locales/pt/draft.json`
- `client/src/pages/DraftPage.tsx`
- `client/src/pages/DraftPodPage.tsx`
- `client/src/pages/__tests__/DraftPage.workspace.test.tsx`
- `client/src/pages/__tests__/DraftPodPage.modeEntry.test.tsx`

## Track

Developer

## LLM

Model: gpt-5.6-sol
Tier: Frontier
Thinking: max

## Implementation method (required)

Method: not-applicable — frontend-only display, localization data, and regression tests

## CR references

- CR 903.13a, CR 903.13b, and CR 903.13f — copy/test documentation only; no rules implementation changed.

## Verification

- [x] Required checks ran clean, or the exact CI-owned alternative is stated below.
- [x] Gate A output below is for the current committed head.
- [x] Final review-impl below is clean for the current committed head.
- [x] Both anchors cite existing analogous code at the same seam.

- `cargo fmt --all` — exit 0; committed worktree remained clean.
- `corepack pnpm@9.15.9 run protocol:check` — exit 0.
- `corepack pnpm@9.15.9 exec tsc -b --noEmit` — exit 0.
- `corepack pnpm@9.15.9 exec eslint .` — exit 0; 0 errors and the repository's 41 existing warnings, none in the changed files.
- Focused Vitest run for the five changed component/page/localization suites — 5 test files passed; 134 tests passed.
- `corepack pnpm@9.15.9 exec vitest run` — 402 test files passed, 2 skipped; 4,193 tests passed, 12 todo.
- `jq -e .` across all seven changed `draft.json` catalogs — exit 0.
- `git diff --check origin/main...HEAD` — exit 0.
- Repository gate scripts — Gate G, Gate A, and Gate P passed after the final review at the committed head.

## Gate A

Gate A PASS head=7d10e28e724e55c67e367c2a5b65bbe4f46b3c65 base=2c42e0f45632aac25a53f2e4b07e9f63d1705928

## Anchored on

- `client/src/components/draft/DraftProgress.tsx:37` — existing draft progress renders the engine-published `pack_count` rather than duplicating a format default.
- `client/src/components/draft/SealedPackOpening.tsx:116` — existing sealed copy routes the engine-derived card count through a CLDR-aware count key.

## Final review-impl

Final review-impl PASS head=7d10e28e724e55c67e367c2a5b65bbe4f46b3c65

## Claimed parse impact

None.

## Scope Expansion

None.

## Validation Failures

None.

## CI Failures

None.

## Summary by CodeRabbit

- **New Features**
  - Draft introductions now show accurate pack counts, pack sizes, minimum deck sizes, and pack-passing directions for Quick, Pod, and Commander drafts.
  - Mixed-size packs are clearly described in drafting instructions.
  - Sealed pack-opening instructions now reflect the actual number of packs.

- **Localization**
  - Updated draft and sealed-opening instructions across supported languages with singular and plural wording.
  - Added dynamic translations for pack quantities, card counts, deck minimums, and pack sizes.

### CODE REVIEW

File: client/src/components/draft/DraftIntro.tsx [hunk 1]
```diff
@@ -30,46 +31,61 @@ interface Step {
...
+  // and players draft two cards from each booster before passing it. Commander
+  // reuses the pod player-count key and the shared pack-passing key rather than
+  // duplicating either sentence into seven locales.
   const commanderSteps: Step[] = [
     { icon: "1", text: t("intro.pod.step1", { count: podSize }) },
     { icon: "2", text: t("intro.commander.step2") },
-    { icon: "3", text: t("intro.pod.step3") },
-    { icon: "4", text: t("intro.commander.step4") },
+    { icon: "3", text: packPassing },
+    { icon: "4", text: t("intro.commander.step4", { minimumDeckCards }) },
```
[the remark this answers is not in the document]
Comment (mwd1234) [reply client/src/components/draft/DraftIntro.tsx:96]: Addressed in 7d10e28e7. Commander step 2 now has uniform and mixed pack-size paths using the engine-published pack count, current card count, and ordered pack sizes across all seven locales. Regression coverage includes a non-default 4 x 18 Commander procedure, a mixed 20/18/20 procedure, and DraftPodPage view wiring. Focused suites pass 134 tests; the full frontend suite passes 4,193 tests, with TypeScript and ESLint clean.

File: client/src/components/draft/DraftIntro.tsx [hunk 2]
```diff
@@ -30,46 +31,61 @@ interface Step {
...
-  // CR 903.13a/b. Steps 1 and 3 are identical to the pod procedure, so they REUSE the
-  // pod keys rather than duplicating two sentences into seven locales — the same reuse
-  // the draft landing page makes for the menu-namespace "Enter" CTA. It also keeps
-  // `{{count}}` on a key whose seven translations already interpolate it correctly.
+  // CR 903.13a/b: Commander Draft is a draft followed by a multiplayer game,
+  // and players draft two cards from each booster before passing it. Commander
+  // reuses the pod player-count key and the shared pack-passing key rather than
+  // duplicating either sentence into seven locales.
   const commanderSteps: Step[] = [
     { icon: "1", text: t("intro.pod.step1", { count: podSize }) },
```
[the remark this answers is not in the document]
Comment (mwd1234) [reply client/src/components/draft/DraftIntro.tsx:85]: Addressed in 7d10e28e7. Commander step 2 now has uniform and mixed pack-size paths using the engine-published pack count, current card count, and ordered pack sizes across all seven locales. Regression coverage includes a non-default 4 x 18 Commander procedure, a mixed 20/18/20 procedure, and DraftPodPage view wiring. Focused suites pass 134 tests; the full frontend suite passes 4,193 tests, with TypeScript and ESLint clean.

File: client/src/components/draft/SealedPackOpening.tsx [hunk 3]
```diff
@@ -190,7 +190,9 @@ export function SealedPackOpening({ view, onComplete }: SealedPackOpeningProps)
       {!showReview && (
         <div className="mb-5 text-center">
           <h1 className="menu-display text-3xl text-white">{t("sealedOpening.title")}</h1>
-          <p className="mt-2 text-sm text-white/50">{t("sealedOpening.subtitle")}</p>
+          <p className="mt-2 text-sm text-white/50">
+            {t("sealedOpening.subtitle", { count: packs.length })}
```
[the remark this answers is not in the document]
Comment (mwd1234) [reply client/src/components/draft/SealedPackOpening.tsx]: Addressed in 7d10e28e7. The sealed subtitle now passes view.pack_count directly, and the singular fixture sets pack_count to 1 alongside its single pack. Payload traversal still uses sealed_packs.length because it indexes that concrete array; the published procedure copy no longer derives its count there. Focused suites pass 134 tests; the full frontend suite passes 4,193 tests.

**Comment by mwd1234** [comment]
@codex review

**Review by matthewevans — APPROVED** [review]
Approved at `7d10e28e724e55c67e367c2a5b65bbe4f46b3c65`: live engine-published draft procedure values are rendered at the appropriate display seam, including Commander, mixed-pack, and localized quantity coverage.
